modules/openapi-generator/src/main/resources/ruby-client/README.mustache

```diff
@@ -66,9 +66,15 @@ require '{{{gemName}}}'
 {{#apiInfo}}{{#apis}}{{#-first}}{{#operations}}{{#operation}}{{#-first}}{{#hasAuthMethods}}
 # Setup authorization
 {{{moduleName}}}.configure do |config|{{#authMethods}}{{#isBasic}}
-  # Configure HTTP basic authorization: {{{name}}}
-  config.username = 'YOUR USERNAME'
-  config.password = 'YOUR PASSWORD'{{/isBasic}}{{#isApiKey}}
+{{^isBasicBearer}}
+# Configure HTTP basic authorization: {{{name}}}
+config.username = 'YOUR_USERNAME'
+  config.password = 'YOUR_PASSWORD'
+{{/isBasicBearer}}
+{{#isBasicBearer}}
+# Configure Bearer authorization{{#bearerFormat}} ({{{.}}}){{/bearerFormat}}: {{{name}}}
+config.access_token = 'YOUR_BEARER_TOKEN'
+{{/isBasicBearer}}{{/isBasic}}{{#isApiKey}}
```

Comment:
@bopm thanks for the enhancements. I wonder if you can indent the code above with additional 2-spaces (except line 72)

Reply:
Done.